hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieCompactionConfig.java

```diff
@@ -166,6 +166,11 @@
       .withDocumentation("Amount of MBs to spend during compaction run for the LogFileSizeBasedCompactionStrategy. "
           + "This value helps bound ingestion latency while compaction is run inline mode.");
 
+  public static final ConfigProperty<Long> COMPACTION_LOG_FILE_SIZE_THRESHOLD = ConfigProperty
+      .key("hoodie.compaction.logfile.size.threshold")
+      .defaultValue(1024 * 1024 * 1024L)
+      .withDocumentation("Only if the log file size is greater than the threshold, the file group will be compacted.");
```

Comment:
nit:than the threshold -> than the threshold in bytes?

Reply:
already fix